app/services/ArrivalMovementMessageService.scala

```diff
@@ -80,10 +77,27 @@ class ArrivalMovementMessageService @Inject()(arrivalIdRepository: ArrivalIdRepo
       xmlMessage <- ReaderT[ParseHandler, NodeSeq, NodeSeq](nodeSeqToEither)
     } yield MovementMessageWithoutStatus(dateTime, messageType, xmlMessage, messageCorrelationId)
 
+  //TODO: Not sure if this is used anymore
   def makeMovementMessageWithStatus(messageCorrelationId: Int, messageType: MessageType): ReaderT[ParseHandler, NodeSeq, MovementMessageWithStatus] =
     for {
       _          <- correctRootNodeR(messageType)
       dateTime   <- dateTimeOfPrepR
       xmlMessage <- ReaderT[ParseHandler, NodeSeq, NodeSeq](nodeSeqToEither)
     } yield MovementMessageWithStatus(dateTime, messageType, xmlMessage, SubmissionPending, messageCorrelationId)
```

Comment:
If it looks like dead code we could get rid of it

Reply:
Removed and renamed the methods to make more sense